pegasus/sites.v3/code.org/public/css/font-awesome.css

```diff
@@ -600,9 +600,6 @@
 .fa-sign-in:before {
   content: "\f090";
 }
-.fa-trophy:before {
-  content: "\f091";
-}
```

Comment:
This is the [Font Awesome icon set](http://fontawesome.io/icons/), I don't think we want to remove this entry 😉.